Tools/uorb_graph/create.py

```diff
@@ -227,22 +224,28 @@ def __init__(self, module_whitelist=[], topic_blacklist=[]):
         special_cases_sub = [(a, re.compile(b), re.compile(c) if c is not None else None, re.compile(d))
                                    for a,b,c,d in special_cases_sub]
 
-        self._subscriptions = PubSub(False, topic_blacklist,
-                [r"\borb_subscribe(_multi|)\b\(([^,)]+)"],
-                special_cases_sub)
+        # note: the source-file-string is pre-processed to remove whitespace -- regexes should ignore whitespace
+        # note:  the regexes should have a capture group '()' otherwise they break downstream code
+        capture_cases_sub = [r"\borb_subscribe(_multi|)\b\(([^,)]+)",
+                             r"(uORB::)Subscription\w+[\{\(](\w+\(\w+)"
```

Comment:
- In most cases initialization is at the same place with the declaration, but it does not have to be, e.g. `uORB::SubscriptionData<vehicle_air_data_s>	_sub_airdata;`. It's worth checking and flagging that as well, so we know what we miss.
- what about arrays like `uORB::Subscription _baro_subs[BARO_COUNT_MAX]`?

Reply:
Hmm, good point, I'll take a look at those.   Since you pasted in the code snippets, would you mind pointing me to the actual test cases, so I can verify against them? 

> In most cases initialization is at the same place with the declaration, but it does not have to be, e.g. uORB::SubscriptionData<vehicle_air_data_s> _sub_airdata;. 
> what about arrays like uORB::Subscription _baro_subs[BARO_COUNT_MAX]?

Reply:
Added this to test case #5.